vm-setup/roles/firewall/tasks/main.yml

```diff
@@ -0,0 +1,94 @@
+- set_fact:
+    rhel8: "{{ ansible_os_family == 'RedHat' and ansible_lsb.major_release|int >= 8 }}"
+
+# Use firewalld for RHEL 8
+- name: "Libvirt TCP ports"
+  firewalld:
+    zone: libvirt
+    port: "{{ item }}/tcp"
+    permanent: yes
+    state: enabled
+  loop: "{{ provisioning_tcp_ports + baremetal_tcp_ports }}"
+  become: true
+  when: rhel8
```

Comment:
This aligns with what we've done historically, but I wonder if we could just use firewalld everywhere?  Or at least for RHEL and CentOS - it's running on my local CentOS 7.6 box

Reply:
This all blows up on me, it seems like libvirt wants to use iptables, there's no libvirt zone on el7. Not sure why, needs more investigation I guess

Reply:
Ah Ok, fair enough - I guess at some point we'll perhaps deprecate centos7 support and just standardize on CentOS/RHEL8